hydra/hydra.py

```diff
@@ -473,55 +491,34 @@ def query_cases(
 
         return self.__get_api("cases/", parameters=query_params)
 
-    def search_cases(
-        self,
-        fields=[],
-        accounts=[],
-        cases=[],
-        sbrGroups=[],
-        ownerSsousername=[],
-        tags=[],
-        cluster_ids=[],
-    ):
+    def search_cases(self, **params):
```

Comment:
This is very nit-picky, but the convention is to name this `kwargs`. Do we want to follow that, or use `params` everywhere else?

Reply:
That's fine. I mostly named it params since it was going to be used to directly update the params value for the request, but I have no qualms changing it. I'll do so in the next commit.

Reply:
Resolving as previously noted I have no issue with this change.